docker/jenkins2-centos/Dockerfile

```diff
@@ -9,7 +11,7 @@ RUN mv /usr/libexec/s2i/run /usr/libexec/s2i/run-base
 COPY ./contrib/jenkins /usr/local/bin
 
 # copy the s2i scripts. s2i expects them to be in /usr/libexec/s2i by default.
-ADD ./contrib/s2i /usr/libexec/s2i
+COPY ./contrib/s2i /usr/libexec/s2i
 
 # make the copied s2i scripts executable
 RUN chmod -R +x /usr/libexec/s2i/*
```

Comment:
Maybe I'm missing something but this one make sense to be joined together with run in line 8?